eng/common/scripts/update-docs-metadata.ps1

```diff
@@ -70,12 +70,15 @@ function GetAdjustedReadmeContent($pkgInfo, $lang){
     }
 
     $fileContent = $pkgInfo.ReadmeContent
-    $foundTitle = ""
 
     # only replace the version if the formatted header can be found
     $titleRegex = "(\# Azure .+? (?:client|plugin|shared) library for (?:JavaScript|Java|Python|\.NET|C))"
-    $fileContent = $pkgInfo.ReadmeContent -replace $titleRegex, "`${1} - Version $($pkgInfo.PackageVersion) `n"
-    
+    $titleRegex = "(\# Azure .+? (?:client|plugin|shared) library for (?:JavaScript|Java|Python|\.NET|C))"
+    $fileTitle = ""
+    if ($pkgInfo.ReadmeContent -match $titleRegex) {
+      $fileContent = $pkgInfo.ReadmeContent -replace $titleRegex, "`${1} - Version $($pkgInfo.PackageVersion) `n"
```

Comment:
I believe we need a fallback for `$fileContent`. What happens if the header doesn't parse appropriately? Currently it'll just put the header as-is, followed by the rest of the content. With this update it'll also clear out the rest of the readme content.

Reply:
$fileContent assigned the value of $pkgInfo.ReadmeContent  in line 72 if not matching